src/main/java/sorinfoca/driver/BrowserManager.java

```diff
@@ -6,9 +6,12 @@
 
 public class BrowserManager {
 
+
+public class BrowserManager {
```

Comment:
If you add these lines, you should have a compilation error as you start to define a class two times in the same file. Please double check this and remove the second class definition from this file. Thanks!